src/devices/Ht16k33/Enums/Address.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ht16k33.Enums
+{
+    /// <summary>
+    /// Digit address
+    /// </summary>
+    /// <remarks>
+    /// Index for the digit value in the display buffer
+    /// </remarks>
+    public enum Address
```

Comment:
does this need to be public?

Reply:
It's for the indexer property in SevenSegment.cs:
https://github.com/dotnet/iot/pull/772/files#diff-912ce972ab785da08d13dd57b875ba50R86

Reply:
Would it make sense to just take int as an input and translate it into this internally?

Reply:
That could work too, I'll do it in my next amend.